File: DownloadFiles/TGNet/Chapter10/VB/Exercise10-1/App_Code/Service.vb
Imports System.Web
Imports System.Web.Services
Imports System.Web.Services.Protocols

<WebService(Namespace := "http://tempuri.org/")> _
<WebServiceBinding(ConformsTo:=WsiProfiles.BasicProfile1_1)> _
Public Class Service
     Inherits System.Web.Services.WebService

    Public Sub Service

    End Sub

    <WebMethod(Description:="This service calculates the area of a rectangle")> _
    Public Function RectangleArea(ByVal length As Single, ByVal width As Single) As Single
        Return length * width
    End Function

End Class

File: DownloadFiles/TGNet/Chapter10/C#/Exercise10-1/App_Code/Service.cs
using System;
using System.Web;
using System.Web.Services;
using System.Web.Services.Protocols;

[WebService(Namespace = "http://tempuri.org/")]
[WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
public class Service : System.Web.Services.WebService
{
    public Service () {

    }

    [WebMethod(Description = "This service calculates the area of a rectangle")]
    public Single RectangleArea(Single length, Single width) {
        return length * width;
    }
    
}
